hedera-mirror-rest/__tests__/specs/schedules-06-specific-id-no-signatures.spec.json

```
@@ -19,9 +19,11 @@
       {
         "consensus_timestamp": "1234567890000000001",
         "creator_account_id": "0.0.1024",
+        "expiration_time": "1234567890000000002",
```

This is a timestamp in the proto, so should be formatted like our other timestamps.
```suggestion
        "expiration_time": "1234567890.000000002",
```

---

Done.